flaskfront/setup.py

```diff
@@ -0,0 +1,20 @@
+from setuptools import setup, find_packages
+
+setup(
+    name="opgflaskfront",
+    version="1.0",
+    description="base flask front app",
+    install_requires=[
```

Comment:
I'd remove anything from here which is not a direct dependency of the library. I would also add a dev section with the pytest dependencies.

Reply:
I already attempted to do precisely that i:e pare down dependencies to bare minimum.  The only one there that looks not to be a direct dependency current is gov-frontend-wtf but that will soon be required as soon as we're doing forms.    Unless I missed something?  re dev section , sure I will add that

Reply:
Have removed forms deps now